docs/pages/index.tsx

```diff
@@ -16,10 +20,12 @@ const Home: NextPage = () => {
         </h1>
 
         <div className={styles.grid}>
-          <a href="#" className={styles.card}>
-            <h2>Documentation &rarr;</h2>
-            <p>In-depth information on CooperTS features and API</p>
-          </a>
+          <Link href="doc/1">
```

Comment:
What is `1`?

Reply:
1 is just a placeholder, it is a dynamic route to the 1 doc